challenges/functions.js

```diff
@@ -6,27 +6,37 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+function consume (parm1, parm2, cb){
+  return cb(parm1, parm2);
+};
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
-
+function add (num1, num2){
+  console.log(num1 + num2);
+};
+function multiply (num1, num2){
+  console.log(num1 * num2);
+};
+function greeting (firstName, lastName){
+  console.log(`Hello ${firstName} ${lastName}, nice to meet you!`);
+};
 
 /* Step 3: Check your work by un-commenting the following calls to consume(): */
-// consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
+consume(2,2,add); // 4
+consume(10,16,multiply); // 160
+consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
```

Comment:
You need to console log `consume()` since it has a return statement in the declaration.